std/string.d

```diff
@@ -3047,16 +3100,71 @@ if (isConvertibleToString!Range)
     cast(void) stripRight(ws.byUTF!wchar).array;
 }
 
+/// Ditto
+auto stripRight(Range)(Range str, string chars)
+if (isBidirectionalRange!Range && isSomeChar!(ElementEncodingType!Range) &&
+    !isInfinite!Range && !isConvertibleToString!Range)
+{
+    while (!str.empty)
+    {
+        auto c = str.back;
+        if (indexOf(chars, c) == -1)
+            break;
+        str.popBack();
+    }
```

Comment:
There's too much pomp and circumstance in this code. It's a simple for loop with no temporaries:

```d
for (; !str.empty; str.popBack)
{
    if (chars.indexOf(str.back) == -1)
        break;
}
```